### DISCUSSION TITLE
plots and farm gui infinite spin icon

### DISCUSSION THREAD

**Discussion by bigd3435** [Chia-Network/chia-blockchain#8409] [Support]
Hi please see the attatched SS this happens on the farm and plots part of the GUI I have attatched the debug log too.

any help would be greatly appreciated.
![chia load](https://user-images.githubusercontent.com/90348936/132567354-8a9ba729-b798-448f-8abd-54b2d2f8dbc8.png)
[debug.log](https://github.com/Chia-Network/chia-blockchain/files/7131181/debug.log)

**Comment by loppefaaret** [comment]
any chance that you are loading the machine with work ? i do see a few errors in the log, that might indicate that the system could potentially be making small errors ?

**Comment by BrandtH22** [comment]
@bigd3435 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
